src/main/java/com/butbetter/storage/model/ProductInformation.java

```diff
@@ -0,0 +1,100 @@
+package com.butbetter.storage.model;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+public class ProductInformation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name="id", insertable = false, updatable = false, nullable = false)
+    private UUID uuid;
+
+    @Column(name = "delivery_time", nullable = false)
+    private Date deliveryTime;
+
+    @Column(name = "amount", nullable = false)
+    private int amount;
+
+    @OneToOne
```

Comment:
What happens when two products have the same address?
Instead use @ManyToOne annotation 
https://thorben-janssen.com/best-practices-many-one-one-many-associations-mappings/

Reply:
One Product have one address?

Reply:
fixed